components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -218,6 +218,23 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
         );
     }
 
+    showPlaceholder = () => {
+        const {channels, draggingState, category, isNewCategory} = this.props;
+
+        if (category.sorting === CategorySorting.Alphabetical ||
+            category.sorting === CategorySorting.Recency ||
+            isNewCategory) {
+            // Always show the placeholder if the channel being dragged is from the current category
+            if (channels.find((channel) => channel.id === draggingState.id)) {
```

Comment:
Oh, so this is why the drop target hitbox for the DMs category (and presumably new categories) was incorrect before?

Reply:
I'm guessing so, I didn't realize there was a still a placeholder left on the original category when the channel left. It wasn't noticeable since it wasn't taking up space, but maybe it needed to be there for `rbd` to work correctly.